### DISCUSSION TITLE
Moes control panel support

### DISCUSSION THREAD

**Discussion by pimlie** [Blackymas/NSPanel_HA_Blueprint#2403] [Q&A]
Any idea whether this project could support the Moes CCP-TY and/or CCP-Z1 devices? The first one looks similar like the NSPanel Pro (as in big screen without buttons) but are not using Android but the _superior LINUX system_ (their words, not mine ;) ). Unfortunately I haven't been able to find any specs that show what type of chip these devices are using. I did find this video which apparently shows the main board of the TY but I can't recognise any of the chips used: https://www.youtube.com/watch?v=NvbCMklktpw

See https://moeshouse.com/products/tuya-smart-center-control-panel-alexa-built-in & https://moeshouse.com/products/tuya-smart-central-control-panel-handheld

**Comment by edwardtfn** [comment]
I don't think those would work.
This project was developed specifically for ESP32 and Nextion display used by the NSPanel. Not even the NSPanel Pro (Android based) would be supported,
